shared/actions/chat.js

```diff
@@ -1010,6 +1073,18 @@ function _unboxedToMessage (message: MessageUnboxed, idx: number, yourName, your
             key: payload.serverHeader.messageID,
             deletedIDs: payload.messageBody.delete && payload.messageBody.delete.messageIDs || [],
           }
+        case CommonMessageType.edit: {
+          const outboxID = payload.clientHeader.outboxID && outboxIDToKey(payload.clientHeader.outboxID)
+          return {
+            key: common.messageID,
+            message: new HiddenString(payload.messageBody && payload.messageBody.edit && payload.messageBody.edit.body || ''),
+            messageID: common.messageID,
+            outboxID,
+            targetMessageID: payload.messageBody.edit ? payload.messageBody.edit.messageID : 0,
```

Comment:
0 itself isn't a valid mesageid so it is effectively a -1. this was more to satisfy flow than anything